server/src/controllers/rsvpController.ts

```diff
@@ -33,7 +46,26 @@ export const deleteRsvp = async (req: Request, res: Response, next: Function) =>
   }
 };
 
+export const updateRsvpByTokenController = async (req: Request, res: Response, next: Function) => {
+  try {
+    const { id: eventId, rsvpId } = req.params;
+    const { token, name, status, guests } = req.body;
+    console.log(req.body, req.params)
+    if (!token || !name || !status === undefined) {
+      return res.status(400).json({ error: "Token, name, status, and guests are required." });
```

Comment:
Also, we mention `guests` as required but we're not checking for that, so I guess we need to add that check 👍